eq-author/public/index.html

```diff
@@ -8,6 +8,21 @@
     <meta http-equiv="expires" content="0" />
     <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
     <meta http-equiv="pragma" content="no-cache" />
+    <script>
+      var isChrome =
+        !!window.chrome &&
+        (!!window.chrome.webstore || !!window.chrome.runtime);
+      function isElectron() {
+        return (
+          typeof process !== "undefined" &&
```

Comment:
A falsy check here is sufficient. Are we finding places where that is not sufficient?

Reply:
Only check if `process` is falsy? Which means it is being run in a browser?

Reply:
I mean just do
`process && process.versions && process.versions.electron`

No need for explicitly checking undefined.

Reply:
The function breaks in IE11 when doing it that way.

Reply:
how? Surely process is undefined so it will shortcut it?

Reply:
I just checked. Its read as a global. You could `window.process && ...`

Reply:
When I run this in IE11:
```js
if(process && process.versions && process.versions.electron) {
  console.log("true")}
else {
  console.log("false")
}
```
It displays an error with `process' is undefined`

While this one logs `false`
```js
if(typeof process !== "undefined" && process.versions && process.versions.electron !== undefined) {
  console.log("true")
} 
else {
  console.log("false")
}
```

Reply:
Yeah I missed it would throw because the `process` variable wouldn't be defined. But as discussed this isn't needed now